assets/src/scripts/block-carousel.js

```diff
@@ -66,8 +133,74 @@ const init = () => {
 
 		domElement.carousel = new Splide( domElement, options ).mount();
 
+		if ( isVideoCarousel ) {
+			// Function to apply proximity-based scaling and opacity to slides
+			const applyProximityScaling = ( centerIndex ) => {
+				const slides = domElement.carousel.Components.Slides.get();
+				const visibleRange = Math.floor( perPage / 2 );
+
+				slides.forEach( ( slide ) => {
+					const index = slide.index;
+					// Calculate distance from center (0 = center, increases with distance)
+					const distance = Math.abs( index - centerIndex );
+
+
+					// Calculate scale: 1.0 at center, decreasing for distant slides
+					// Using a linear interpolation
+					const maxDistance = visibleRange; // Adjust based on perPage
+					const minScale = 0.6;
+					const maxScale = 1.0;
+					const normalizedDistance = Math.min( distance / maxDistance, 1 );
+					const scale = maxScale - (normalizedDistance * (maxScale - minScale));
+
+					// Calculate opacity: 1.0 at center, decreasing for distant slides
+					const minOpacity = 0.1;
+					const maxOpacity = 1.0;
+					const opacity = maxOpacity - (normalizedDistance * (maxOpacity - minOpacity));
+
+
+					// Apply transform, opacity, and padding directly
+					if ( slide.slide.classList.contains( 'wp-block-shiro-video-promo-container' ) ) {
+						// Apply scale to all video and iframe children
+						const videos = slide.slide.querySelectorAll( 'video' );
+						const iframes = slide.slide.querySelectorAll( 'iframe' );
+						videos.forEach( video => {
+							video.style.transform = `scale(${scale})`;
+						} );
+						iframes.forEach( iframe => {
+							iframe.style.transform = `scale(${scale})`;
+						} );
+					} else {
+						slide.slide.firstChild.style.transform = `scale(${scale})`;
+					}
+					slide.slide.style.opacity = "" + opacity;
```

Comment:
Is the ""  doing anything here?

Reply:
Opacity is a float and this converts it into a string.